src/Home/RankingItem/ProductList.tsx

```diff
@@ -0,0 +1,198 @@
+  type: string;
+  imageSrc: string;
+  title: string;
+  subtitle: string;
+  amount: number;
+}
+
+const products: Product[] = [
+  {
+    id: 1,
+    category: 'women',
+    type: 'wanted',
+    imageSrc: '/images/chicken.png',
+    title: 'BBQ 양념치킨+크림치즈볼+콜라1.25L',
+    subtitle: 'Delicious Chicken',
+    amount: 29000,
+  },
+  {
+    id: 2,
+    category: 'men',
+    type: 'gifted',
+    imageSrc: '/images/chicken.png',
+    title: 'BBQ 양념치킨+크림치즈볼+콜라1.25L',
+    subtitle: 'Tasty Chicken',
+    amount: 29000,
+  },
+  {
+    id: 3,
+    category: 'teens',
+    type: 'wishlist',
+    imageSrc: '/images/chicken.png',
+    title: 'BBQ 양념치킨+크림치즈볼+콜라1.25L',
+    subtitle: 'Tasty Chicken',
+    amount: 29000,
+  },
+  {
+    id: 4,
+    category: 'women',
+    type: 'wanted',
+    imageSrc: '/images/chicken.png',
+    title: 'BBQ 양념치킨+크림치즈볼+콜라1.25L',
+    subtitle: 'Delicious Chicken',
+    amount: 29000,
+  },
+  {
+    id: 5,
+    category: 'men',
+    type: 'gifted',
+    imageSrc: '/images/chicken.png',
+    title: 'BBQ 양념치킨+크림치즈볼+콜라1.25L',
+    subtitle: 'Tasty Chicken',
+    amount: 29000,
+  },
+  {
+    id: 6,
+    category: 'teens',
+    type: 'wishlist',
+    imageSrc: '/images/chicken.png',
+    title: 'BBQ 양념치킨+크림치즈볼+콜라1.25L',
+    subtitle: 'Tasty Chicken',
+    amount: 29000,
+  },
+  {
+    id: 7,
+    category: 'women',
+    type: 'wanted',
+    imageSrc: '/images/chicken.png',
+    title: 'BBQ 양념치킨+크림치즈볼+콜라1.25L',
+    subtitle: 'Delicious Chicken',
+    amount: 29000,
+  },
+  {
+    id: 8,
+    category: 'men',
+    type: 'gifted',
+    imageSrc: '/images/chicken.png',
+    title: 'BBQ 양념치킨+크림치즈볼+콜라1.25L',
+    subtitle: 'Tasty Chicken',
+    amount: 29000,
+  },
+  {
+    id: 9,
+    category: 'women',
+    type: 'wanted',
+    imageSrc: '/images/chicken.png',
+    title: 'BBQ 양념치킨+크림치즈볼+콜라1.25L',
+    subtitle: 'Delicious Chicken',
+    amount: 29000,
+  },
+  {
+    id: 10,
+    category: 'women',
+    type: 'wanted',
+    imageSrc: '/images/chicken.png',
+    title: 'BBQ 양념치킨+크림치즈볼+콜라1.25L',
+    subtitle: 'Delicious Chicken',
+    amount: 29000,
+  },
+  {
+    id: 11,
+    category: 'women',
+    type: 'wanted',
+    imageSrc: '/images/chicken.png',
+    title: 'BBQ 양념치킨+크림치즈볼+콜라1.25L',
+    subtitle: 'Delicious Chicken',
+    amount: 29000,
+  },
+  {
+    id: 12,
+    category: 'women',
+    type: 'wanted',
+    imageSrc: '/images/chicken.png',
+    title: 'BBQ 양념치킨+크림치즈볼+콜라1.25L',
+    subtitle: 'Delicious Chicken',
+    amount: 29000,
+  },
+  {
+    id: 13,
+    category: 'women',
+    type: 'wanted',
+    imageSrc: '/images/chicken.png',
+    title: 'BBQ 양념치킨+크림치즈볼+콜라1.25L',
+    subtitle: 'Delicious Chicken',
+    amount: 29000,
+  },
+];
+
+const ProductList = ({
+  activeCategory,
+  activeType,
+}: {
+  activeCategory: string;
+  activeType: string;
+}) => {
+  const [visibleProducts, setVisibleProducts] = useState<number>(6);
+  const [filteredProducts, setFilteredProducts] = useState<Product[]>([]);
```

Comment:
[useEffect 내부에서 처리하는 것이](https://github.com/kakao-tech-campus-2nd-step2/react-gift-ui-flow/pull/51/files#diff-165878b8c9a04b64a5ce28fa82778f724364c8fa210486cd811f99a74b0eefb2R149-R153) 아니라 필터 함수를 작성해서 `filteredProducts` 초기값을 아래와 같이 지정해 줄 수 있습니다.

```suggestion
  const [filteredProducts, setFilteredProducts] = useState<Product[]>(() => products.filter(
      (product) =>
        (activeCategory === 'all' || product.category === activeCategory) &&
        product.type === activeType,
    ));
```